.github/workflows/wheel_tests_and_release.yml

```diff
@@ -115,9 +119,19 @@ jobs:
       - uses: actions/setup-python@v5
         name: Install Python
         with:
-          python-version: "3.9"
+          python-version: "3.12"
+
+      - name: Build wheels for CPython (macOS) (x86_64)
+        if: matrix.cibw_arch == 'x86_64' && matrix.os == 'macos-latest'
```

Comment:
The `&& matrix.os == 'macos-latest'` doesn't seem needed, just checking for `x86_64` is enough.

Reply:
Addressed in 868515d5e0ac1f8038c910415a754ece5189b82a.